### PR TITLE
new look and feel

### PR DESCRIPTION
**Pull request by jcichoski**
Unfortunately I do not know how to test my change, if you like my attempt feel free to take it and make it work. IMHO this change will make the wiki page more user friendly when navigating through resources. This change removes the need for the anchor links as the resources themselves are tucked away in expandable drop downs.

### CODE REVIEW

**Comment by jhitchcock** [comment]
Thanks. I'll try get this sorted this week :)

**Comment by jcichoski** [comment]
Hi James, have you had a chance to take a look at this change? Is there anything I can do, or anything it needs I can help out with?

**Comment by jhitchcock** [comment]
Hey sorry for the delay. I'm going to extract this to an alternative template, and have a parameter to pick a template with the existing as default, since people may already wish to stick with the initial format or not wish it to change when they upgrade. 

I'll do this today! :)

**Comment by jhitchcock** [comment]
I've implemented this finally, thanks for your contribution! 

https://github.com/jhitchcock/raml2wiki/commit/171b17962650c02139a21487c24965c6b24ead13

File: lib/template.handlebars [hunk 1]
```diff
@@ -44,12 +39,9 @@ h3. Uri Parameters
 
 {{/if}}
 
-
-
+{expand}
```
Comment (jhitchcock) [inline lib/template.handlebars]: This should be below the actual resource include

File: lib/template.handlebars [hunk 2]
```diff
@@ -19,16 +16,14 @@ h4. Base Uri Parameters
...
 {{#documentation}}
 h3. {{{title}}}
 {{{content}}}
 {{/documentation}}
 
-
-
 {{#resources}}
-h2. {{#if displayName}}{{{wikisafe displayName}}}{{/if}}{{#unless displayName}}{{{wikisafe relativeUri}}}{{/unless}}
+h2. Resources:
```
Comment (jhitchcock) [inline lib/template.handlebars]: This should be above {{#resources}} or it will repeat. Sorted

**Pull request closed without merging**